pkg/cri/store/image/image.go

```diff
@@ -91,6 +92,20 @@ func (s *Store) Update(ctx context.Context, ref string) error {
 // update updates the internal cache. img == nil means that
 // the image does not exist in containerd.
 func (s *Store) update(ref string, img *Image) error {
+	refName := strings.Split(ref, ":")[0]
+	switch refName {
+	case string(imagedigest.SHA256), string(imagedigest.SHA512), string(imagedigest.SHA384):
+		//  a digested reference
+		break
```

Comment:
golang doesnt fallthrough, so no need to break.

Reply:
Fixed.